src/features/user/Account/History.tsx

```diff
@@ -68,20 +126,33 @@ export default function History({
       <div className={'profilePageSubTitle'}>{t('me:donationsSubTitle')}</div>
       <div className={styles.pageContainer}>
         <div className={styles.filterRow}>
-          {accountingFilters &&
-            Object.entries(accountingFilters).map((item) => {
-              return (
-                <div
-                  key={item[0]}
-                  className={`${styles.filterButton} ${
-                    filter === item[0] ? styles.selected : ''
-                  }`}
-                  onClick={() => handleSetFilter(item[0])}
-                >
-                  {t(item[0])}
-                </div>
-              );
-            })}
+          {accountingFilters && (
+            <>
+              {Object.entries(accountingFilters).map((item) => {
+                return (
+                  <div
+                    key={item[0]}
+                    className={`${styles.filterButton} ${
+                      filter === item[0] ? styles.selected : ''
+                    }`}
+                    onClick={() => handleSetFilter(item[0])}
+                  >
+                    {t(item[0])}
+                  </div>
+                );
+              })}
+            </>
```

Comment:
Any reason we're introducing the `<></>` tags here? They don't seem necessary.

Reply:
I added some logic and then I removed the logic, forgot to remove the fragments😅

Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/3dbbcc600c81cbcc313645b973754c9423c870c5